### PR TITLE
Log level adjustments

### PR DESCRIPTION
**Pull request by akallabeth**
* Reduced log level for some messages

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot Be nice and test this through.

File: libfreerdp/core/rdp.c [hunk 1]
```diff
@@ -363,7 +363,7 @@ BOOL rdp_read_header(rdpRdp* rdp, wStream* s, UINT16* length, UINT16* channelId)
 				rdp_set_error_info(rdp, ERRINFO_RPC_INITIATED_DISCONNECT);
 		}
 
-		WLog_ERR(TAG, "DisconnectProviderUltimatum: reason: %d", reason);
+		WLog_DBG(TAG, "DisconnectProviderUltimatum: reason: %d", reason);
```
Comment (bmiklautz) [inline libfreerdp/core/rdp.c]: Is there now a message printed in those cases? If not we should maybe change this to info.

Comment (akallabeth) [inline libfreerdp/core/rdp.c]: @bmiklautz Yes, the error info is always printed. This message did just print the more or less cryptic integer version of it.

**Comment by mfleisz** [comment]
+1

**Comment by bmiklautz** [comment]
+1

**Pull request merged**